packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -107,50 +104,73 @@ const ActiveVisitsTable = () => {
     [t],
   );
 
-  const rowData = activeVisits.map((visit) => ({
-    ...visit,
-    visitStartTime: formatDatetime(parseDate(visit.visitStartTime)),
-  }));
-
   const searchResults = useMemo(() => {
     if (searchString && searchString.trim() !== '') {
       const search = searchString.toLowerCase();
-      return rowData.filter((activeVisitRow) =>
-        Object.keys(activeVisitRow).some((header) => {
+      return activeVisits.filter((activeVisitRow) =>
+        Object.entries(activeVisitRow).some(([header, value]) => {
           if (header === 'patientUuid') {
             return false;
           }
-          return `${activeVisitRow[header]}`.toLowerCase().includes(search);
+          return `${value}`.toLowerCase().includes(search);
         }),
       );
-    } else {
-      return rowData;
     }
-  }, [searchString, rowData]);
 
-  const {
-    goTo,
-    results: paginatedActiveVisits,
-    currentPage,
-  }: PaginationData = usePagination(searchResults, currentPageSize);
+    return activeVisits;
+  }, [searchString, activeVisits]);
 
-  const handleSearch = useCallback((e) => setSearchString(e.target.value), []);
+  const { paginated, goTo, results, totalPages, currentPage } = usePagination(searchResults, pageSize);
```

Comment:
The `totalPages` is not used anywhere, we could just remove it.

Reply:
Sure!
Thanks!